test/fixtures/files/upload_for_release_in_name_other.txt

```diff
@@ -0,0 +1 @@
+Upload file content
```

Comment:
Why adding this file? I forces to modify a bunch of unrelated tests that read the whole directory.

Can you just write one additional test that use a variable for the name and label?